src/DebugSql.php

```diff
@@ -77,7 +77,9 @@ public static function sql(Query $query, $showValues = true, $showHtml = null, $
         }
 
         $trace = Debugger::trace(['start' => 1, 'depth' => $stackDepth + 2, 'format' => 'array']);
+        /** @psalm-suppress PossiblyInvalidArrayOffset */
         $file = isset($trace[$stackDepth]) ? $trace[$stackDepth]['file'] : 'n/a';
```

Comment:
I'm not really sure why psalm thinks these should be int offsets.

Reply:
This could be simplified to 
```suggestion
        $file = $trace[$stackDepth]['file'] ?? 'n/a';
```